packages/bpk-component-theme-toggle/src/theming.js

```diff
@@ -119,7 +125,7 @@ const bpkCustomThemes = {
     primaryColor500: '#006463',
     primaryColor600: '#024D4D',
     primaryColor700: '#013838',
-    secondaryColor300: '#686868',
```

Comment:
@jamesf3rguson do you wanna weigh in on this? I assume we wanna avoid changing the theme if we can

Reply:
to give you some background: `secondaryColor300` wasn't used anywhere and I needed a lighter color for the `story` example.